packages/client/wallets/aa/src/utils/user.ts

```diff
@@ -0,0 +1,75 @@
+import { UserIdentifier, UserIdentifierParams } from "@/types/Config";
+import { isEmpty } from "@/utils/helpers";
+import EmailValidator from "email-validator";
+import { parsePhoneNumber } from "libphonenumber-js";
+
+export function narrowUserIdentifier(userIdentifier: UserIdentifier) {
```

Comment:
is this needed? I think it's ok to have extra params in either case, whether it's a log or an API call, I'm not worried about including the `type`

Reply:
Yes it's needed, but we can discuss it.